slam_toolbox/config/mapper_params_online_sync.yaml

```diff
@@ -7,10 +7,10 @@ ceres_dogleg_type: TRADITIONAL_DOGLEG
 ceres_loss_function: None
 
 # ROS Parameters
-odom_frame: odom
 map_frame: map
-base_frame: base_footprint
-scan_topic: /scan
+odom_frames: ["robot1/odom", "robot2/odom"]
```

Comment:
Make these changes to all the yaml files so that they all work in this branch with the change of types

Reply:
I updated the yaml for sync and async; I'm not currently sure what would be the best way to handle localization and life-long, since they're very different algorithms; what're your thoughts on this?